frontend/src/components/Entity.vue

```diff
@@ -64,13 +71,14 @@ export default {
     bindArgs() {
       return [
         { schema: this.activeSchema, entity: this.entity },
+        { schema: this.activeSchema, attributes: this.entity },
```

Comment:
The entity is used for editing and it does different requests, that's why I created an additional property for attributes. If there is no entity, the EntityForm automatically sets the editEntity attributes to null. It's probably a little bit confusing here, since I'm using the whole entity.. 

I can try to refactor a little bit, so the EntityForm can send an update request only if there is entity id and remove the attributes property. If I go this direction we will still need another property for conditional displaying of the attribute checkboxes. But I guess this will be more readable and it will make more sense.

Reply:
I think I need another look at the code without getting interrupted.

If you think this will increase readability and if you are willing to refactor like that, it will be much appreciated :slightly_smiling_face:

Reply:
> The entity is used for editing and it does different requests, that's why I created an additional property for attributes.

Hey, sorry, I want correct myself a bit - today, when I went through the code, I realized that the above explanation which I gave is not relevant since I use directly createEntity() function from EntityForm component..